libs/core/src/integration/events.schemas.ts

```diff
@@ -173,6 +177,14 @@ export const ChainEventCreated = z.union([
     }),
     parsedArgs: CommunityStakeTrade,
   }),
+  ChainEventCreatedBase.extend({
+    eventSource: ChainEventCreatedBase.shape.eventSource.extend({
+      eventSignature: z.literal(
+        '0xd7ca5dc2f8c6bb37c3a4de2a81499b25f8ca8bbb3082010244fe747077d0f6cc',
```

Comment:
is this the same as the signature? should we move signatures to libs/core or some other restructuring so we can avoid repetition?

Reply:
I agree. There is a lot of repetition here, the idea is that this is a quick and dirty implementation for when we perform the refactor talked about last friday.